packages/react-core/src/components/Page/Page.tsx

```diff
@@ -197,6 +210,36 @@ export class Page extends React.Component<PageProps, PageState> {
       isNavOpen: mobileView ? mobileIsNavOpen : desktopIsNavOpen
     };
 
+    const nav = tertiaryNav ? (
+      isTertiaryNavWidthLimited ? (
+        <div className={css(styles.pageMainNav, styles.modifiers.limitWidth)}>
+          <div className={css(styles.pageMainBody)}>{tertiaryNav}</div>
+        </div>
+      ) : (
+        <div className={css(styles.pageMainNav)}>{tertiaryNav}</div>
+      )
+    ) : null;
+
+    const crumb = breadcrumb ? (
+      isBreadcrumbWidthLimited ? (
+        <section className={css(styles.pageMainBreadcrumb, styles.modifiers.limitWidth)}>
+          <div className={css(styles.pageMainBody)}>{breadcrumb}</div>
+        </section>
+      ) : (
+        <section className={css(styles.pageMainBreadcrumb)}>{breadcrumb}</section>
+      )
+    ) : null;
+
+    const isGrouped = isTertiaryNavGrouped || isBreadcrumbGrouped;
+
+    const group = (
```

Comment:
not sure about performance impact.. but you could add a check `isGrouped` and if false return null

Reply:
For `group`? Sure I'll add it.